spec/unit/facter/java_patch_level_spec.rb

```diff
@@ -17,8 +17,8 @@
       end
     end
 
-    context "if java is installed" do
-      context 'returns java patch version extracted from java_version fact' do
+    context "if java is not installed" do
+      context 'returns nil because of non existent java_version fact' do
```

Comment:
Ah, gotcha.

Improved wording on that second line might just be be `context 'returns nil' do`.